app/views/clones/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h2>Clone of Project <%= @clone.project.name %></h2>
+<%= link_to 'View your repo on GitHub', @clone.url %>
+<p>Students working on this project: <%= @clone.students %></p>
+<p>Status: <%= @clone.message %></p>
```

Comment:
It'd be nice to show the students what the different steps are and how far along it is in the process... not saying we have to do that here, but maybe as a future feature?

Reply:
Yes, I think that would be great to add in to a future feature. Maybe a chance to add some cool JS? 😄